src/DibiReflection/DibiReflection.php

```diff
@@ -8,19 +8,24 @@ final class DibiReflection
 {
     public function rewriteQuery(string $queryString): ?string
     {
-        $queryString = str_replace('%in', '(1)', $queryString);
         $queryString = str_replace('%lmt', 'LIMIT 1', $queryString);
         $queryString = str_replace('%ofs', ', 1', $queryString);
-        $queryString = preg_replace('#%(i|s)#', "'1'", $queryString) ?? '';
-        $queryString = preg_replace('#%(t|d)#', '"2000-1-1"', $queryString) ?? '';
+        $queryString = str_replace('%in', '(1)', $queryString);
```

Comment:
```suggestion
        // see https://dibiphp.com/en/documentation#toc-modifiers
        $queryString = str_replace('%in', '(1)', $queryString);
```

Reply:
What do you mean? that `%in` isnt in the table you linked? Its in next table https://dibiphp.com/en/documentation#toc-modifiers-for-arrays

Reply:
added link